src/doc/stdmetadata.rst

```diff
@@ -284,14 +284,94 @@ The following metadata items are specific to photos or captured images.
 
     For captured or scanned image, the model of the camera or scanner.
 
+.. option:: "Focus" : float
+
+    The camera's focus distance of the captured image.
+
 .. option:: "ExposureTime" : float
 
     The exposure time (in seconds) of the captured image.
 
+.. option:: "Iso" : float
+
+    The ISO speed of the captured image.
+
+.. option:: "TStop" : float
+
+    The stop of the camera when it captured the image.
+
 .. option:: "FNumber" : float
 
     The f/stop of the camera when it captured the image.
 
+.. option:: "openexr:CameraTemperature" : float
+
+    Color temperature in Kelvin of the camera when it captured the image.
+
+.. option:: "openexr:CameraTint" : float
+
+    Green/magenta tint of the camera when it captured the image.
+
```

Comment:
Why are these, alone, prefixed as "openexr:"?

Usually, we use the format name as a prefix for one of three reasons:

* Things that are "pseudo-metadata" -- they look like metadata, but actually they are *controls* or *hints* for the writer, information about how the file is written. An example of that is "openexr:roundingmode" that controls how MIP map resolution levels are computed, but isn't really considered the usual token/value style of arbitrary metadata. 
* Items that are so incredibly specific to one format that it makes no sense to try to allow the metadata to copy from one file type to another. Each writer tends to completely ignore metadata that is prefixed by the name of any other format, so that those particular items are never propagated when converting file formats.
* Items where the format-specific way to handle that item is radically different than all the other formats, and we just don't want them mixing.

Reply:
done.